go/registry/api/runtime.go

```diff
@@ -69,6 +73,63 @@ func (k *RuntimeKind) FromString(str string) error {
 	return nil
 }
 
+// ComputeParameters are parameters for the compute committee.
+type ComputeParameters struct {
+	// GroupSize is the size of the committee.
+	GroupSize uint64 `json:"group_size"`
+
+	// GroupBackupSize is the size of the discrepancy resolution group.
+	GroupBackupSize uint64 `json:"group_backup_size"`
+
+	// AllowedStragglers is the number of allowed stragglers.
+	AllowedStragglers uint64 `json:"allowed_stragglers"`
+
+	// RoundTimeout is the round timeout of the nodes in the group.
+	RoundTimeout time.Duration `json:"round_timeout"`
```

Comment:
I would change these timeout values to be `uint64` and always denominated in seconds. Less than second granularities could cause problems so let's just avoid them.

Reply:
I'm not sure that's a good idea since we use `time.Duration` all over the code for storing the duration (all bunch of timeouts and intervals). It's also handy for string parsing (e.g. "10s", "1m5s"). If so, I would rather check whether the granularity passed is a second or more, if that's ok?

Reply:
Yeah adding sanity checks would also be fine.